packages/typespec-test/test/healthInsights_radiologyinsights/generated/typespec-ts/src/models/models.ts

```diff
@@ -1069,49 +1067,6 @@ export function patientRecordArraySerializer(
   });
 }
 
-/** Provides status details for long running operations. */
-export interface HealthInsightsOperationStatusError {
-  /** The unique ID of the operation. */
```

Comment:
pls help investigate why the model is removed.

Reply:
This is because tcgc changes the operation status as LroInitial & Json. I think we could accept this breaking considering  we would not directly use these models. If we have the usage for this model we could add usage decorator to mitigate this breaking.
![image](https://github.com/user-attachments/assets/60d06f61-04be-4dce-8234-50430ece9794)

Reply:
Is this a breaking for HLC? if so, could you create an issue in https://github.com/Azure/autorest.typescript/issues/2367

Reply:
I would assume no. Swagger side if the polling operation is exposed to public then relevant models would be tagged with output usage and would include in modular also. Another thing is the health insight case in which we have nested result in body is not supported in hlc yet.